registration/src/main/proto/lobby/registration/conference.proto

```diff
@@ -70,10 +74,12 @@ message ConferenceAccessCode {
     string value = 1;
```

Comment:
No, the access code is some kind of a password used by a business customer (conference owner) to view conference details. Slug is the part of the URL to access the conference for everybody. I've removed slug because it was defined in the Conference definition in the Conference BC only.